esmvaltool/diag_scripts/ensclus/ens_anom.py

```diff
@@ -0,0 +1,165 @@
+"""Computation of ensemble anomalies based on a desired value."""
+
+# Standard packages
+import sys
+import os
+import numpy as np
+from scipy import stats
+
+
+def ens_anom(filenames, dir_output, name_outputs, varname, numens, season,
+             area, extreme):
+    """Ensemble anomalies.
+
+    Computation of the ensemble anomalies based on the desired value
+    from the input variable (it can be the percentile, mean, maximum, standard
+    deviation or trend)
+    OUTPUT: NetCDF files of ensemble mean of climatology, selected value and
+    anomaly maps.
+    """
+    # User-defined packages
+    from read_netcdf import read3Dncfield, save_N_2Dfields
```

Comment:
Please move imports to the top of the module

Reply:
Done in https://github.com/ESMValGroup/ESMValTool/pull/736/commits/8db45a33ecbb3a12eaaf4b5f1764936e7a951c37 with the help of isort